api/jobs_test.go

```diff
@@ -47,6 +47,43 @@ func TestJobs_Register(t *testing.T) {
 	}
 }
 
+func TestJobs_Parse(t *testing.T) {
+	t.Parallel()
+	c, s := makeClient(t, nil, nil)
+	defer s.Stop()
+
+	jobs := c.Jobs()
+
+	checkJob := func(job *Job, expectedRegion string) {
+		if job == nil {
+			t.Fatal("job should not be nil")
+		}
+
+		region := job.Region
+
+		if region == nil {
+			if expectedRegion != "" {
```

Comment:
In the future feel free to use [`require.NotEmpty`](https://godoc.org/github.com/stretchr/testify/require#NotEmpty) to condense these simple test assertions.

testify is kind of our blessed assertion utility library.

No need to change this PR.